pkg/tsdb/cloudwatch/resource_handler.go

```diff
@@ -16,13 +16,12 @@ func (e *cloudWatchExecutor) newResourceMux() *http.ServeMux {
 	mux := http.NewServeMux()
 	mux.HandleFunc("/regions", handleResourceReq(e.handleGetRegions))
 	mux.HandleFunc("/namespaces", handleResourceReq(e.handleGetNamespaces))
-	mux.HandleFunc("/metrics", handleResourceReq(e.handleGetMetrics))
-	mux.HandleFunc("/all-metrics", handleResourceReq(e.handleGetAllMetrics))
 	mux.HandleFunc("/ebs-volume-ids", handleResourceReq(e.handleGetEbsVolumeIds))
 	mux.HandleFunc("/ec2-instance-attribute", handleResourceReq(e.handleGetEc2InstanceAttribute))
 	mux.HandleFunc("/resource-arns", handleResourceReq(e.handleGetResourceArns))
 	mux.HandleFunc("/log-groups", handleResourceReq(e.handleGetLogGroups))
 	mux.HandleFunc("/all-log-groups", handleResourceReq(e.handleGetAllLogGroups))
+	mux.HandleFunc("/metrics", routes.ResourceRequestMiddleware(routes.MetricsHandler, e.getClients))
```

Comment:
really like the idea that we could move alway from things like "all-metrics" 🎉  we should do the same with logs haha